sys/libkern/builtins/Makefile

```diff
@@ -0,0 +1,11 @@
+# vim: tabstop=8 shiftwidth=8 noexpandtab:
```

Comment:
Is that just a comment or a note to the reader?

Reply:
This is a standard Vim editor annotation for specifying extra commands to be executed at the time file is being open. Please read more here: https://vim.fandom.com/wiki/Modeline_magic